### DISCUSSION TITLE
The DJ management panel is missing. (WEB DJ)

### DISCUSSION THREAD

**Discussion by marbi** [AzuraCast/AzuraCast#7645] [General]
HI,
After activating DJ streaming and adding a new DJ, the Web DJ option does not appear. Could it be that this feature is not available in my version?
Regards
![Obraz 30 12 2024, godz  08 34](https://github.com/user-attachments/assets/29f9c70d-eddc-46e8-bc91-20d7bb2ea9c2)
![Obraz 30 12 2024, godz  08 37](https://github.com/user-attachments/assets/febaa42e-e77c-4eee-aa71-02e44393e8e4)

**Comment by DJMic** [comment]
Very strange, I'm using the same version and the Web DJ Option appear correctly.

**Comment by marbi** [comment]
Thank you for your response!
Is the Web DJ option visible by default after installation, or does it need to be explicitly enabled?
Also, where can I find the logs to investigate further why the Web DJ option is not available?
Best regards, Radek

**Comment by DJMic** [comment]
Do you have you restart your station ? It's needed.

**Reply by marbi** [reply]
I even rebooted the VPS, but it didn't change anything—I still can't see the Web DJ! I also changed the language to English and checked the Icecast error logs, but I didn't notice anything unusual.
Regards
![Obraz 30 12 2024, godz  14 47](https://github.com/user-attachments/assets/e52df9f8-7e0a-4a23-b759-6a0cbbca17c4)

**Comment by marbi** [comment]
OMG! It turns out Public Pages must be ENABLED for Web DJ to work!
Regards
Radek
